Controller/ResourceController.php

```diff
@@ -39,11 +45,104 @@ public function __construct(
         $this->registry = $registry;
     }
 
-    public function resourceAction($repositoryName, $path)
+    public function getResourceAction($repositoryName, $path)
     {
         $repository = $this->registry->get($repositoryName);
         $resource = $repository->get('/'.$path);
 
+        return $this->createResponse($resource);
+    }
+
+    /**
+     * There are two different changes that can currently be done on a cmf resource:
+     *
+     * - move
+     * - rename
```

Comment:
can move only change the parents but not the name itself? does rename allow to only specify the new name but not the path? not sure if that would make worth it to have two different operations. you still need to know the full identifier of the resource.

or is this a puli thing or otherwise required to be two different operations?